task09-reactjs/task09/src/components/General-info.js

```diff
@@ -0,0 +1,128 @@
+                    {info && info.characters.map((item, index)=>(
+                            index < 4 && (
+                                <div className="characters-photo-item">
+                                    <div className="photo-filter"></div>
+                                    <img src={require(`../images/${item.img}`)} className="characters-photo" alt={item.name} />
+                                    <p className="characters-photo-text">{item.name}</p>`;
+                                </div>
+                            )
+                        ))
+                    }
+                </div>
+            </div>
+            <div className="islands-container">
+                <div className="titles-container">
+                    <p className="section-titles">Islands</p>
+                    <a href="/islands" className="see-all-titles">See all</a>
+                </div>
+                <div className="islands-photo-frames" id="islands-photo-frames">
+                    {info && info.islands.map((item, index)=>(
+                        index < 3 && (
+                            <div className="islands-photo-item">
+                                <img src={require(`../images/${item.img}`)} className="img-island" alt={item.name} />
+                                <div className="island-text-frame">
+                                    <p className="island-title">{item.name}</p>
+                                    <div className="island-location">
+                                    <i className="icon-map-marker"></i>
+                                    <p className="island-text">{item.Location}</p>
+                                    </div>
+                                </div>
+                            </div>
+                        )
+                    ))}
+                </div>
+            </div>
+            <div className="mist-obj-container">
+                <div className="titles-container">
+                    <p className="section-titles">Mist Objects</p>
+                    <a href="/mist-objects" className="see-all-titles">See all</a>
+                </div>
+                <div className="objects-photo-container" id="object-photo-container">
+                   {info && info.mysticObjects.map((item, index) => (
+                    index < 4 && (
+                        <div className="objects-photo-item" id={index === 3 ? 'last-obj-item' : ''} >
+                            <img src={require("../images/mist-object.png")} className="obj-img" alt="Mystic Object" />
+                            <div className="object-text-frame">
+                                <p className="mist-obj-title">{item.name}</p>
+                                <p className="mist-obj-text">{item.description}</p>
+                            </div>
+                        </div>
+                    )
+                   ))}
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
Please check the characters,islands and objects scroll specially in mobile version

Reply:
Done, I added the scroll to all 3 :)